csrc/executor.h

```diff
@@ -179,26 +179,89 @@ class FusionExecutor : public NonCopyable {
     return measure_kernel_time_ ? kernel_time_ms_ : 0;
   }
 
+  //! Returns the input bytes accessed for a kernel
+  //! \note It is important to sample the args struct prior to adding the
+  // 1    output to the args struct
+  int64_t inputBytesProcessed(KernelArgumentHolder& args) {
+    int64_t total_bytes = 0;
+    if (!bytes_processed_per_input_.has_value()) {
+      int64_t num_bytes = 0;
+      bytes_processed_per_input_ = std::optional<std::vector<int64_t>>(
```

Comment:
I think it's fine to say `bytes_processed_per_input_ = std::vector<int64_t>(args.size(), 0);`, which is less verbose.

Reply:
Made the change.